js/script.js

```diff
@@ -333,19 +333,20 @@ start_button.addEventListener("click", function () {
   // Timer
 });
 
-i = 60;
+let i = 60; 
+let timeout; 
+let stopTime = -1;
+
 function onTimer() {
   document.getElementById("countdown").innerHTML = i;
   i--;
-  if (i < 0) {
-    clearInterval(i);
-    if (i === 0) {
-      alert("Game Over!");
-    }
-  } else {
-    setTimeout(onTimer, 1000);
+  timeout = setTimeout(onTimer, 1000);
+  if (i <= stopTime) {
+    clearTimeout(timeout);
+    setTimeout(() => window.open('http://127.0.0.1:5505/game%20over.html'), 1000);
```

Comment:
Is this a placeholder page? 
![Screen Shot 2022-10-12 at 4 28 52 PM](https://user-images.githubusercontent.com/93556334/195466180-e3342f89-8a00-45db-9d3e-4999b49972b7.png)

Reply:
no it should be working but I barely realized i git stashed the game over HTML fixing that right now